src/libstd/net/ip.rs

```diff
@@ -59,6 +59,66 @@ pub enum Ipv6MulticastScope {
     Global
 }
 
+impl IpAddr {
+    /// Returns true for the special 'unspecified' address ([IPv4], [IPv6]).
+    /// [IPv4]: ../../std/net/struct.Ipv4Addr.html#method.is_unspecified
+    /// [IPv6]: ../../std/net/struct.Ipv6Addr.html#method.is_unspecified
+    #[unstable(feature="ip", issue="27709",
+               reason="recently added and depends on unstable Ipv4Addr.is_unspecified()")]
```

Comment:
Per [#27709 (comment)](https://github.com/rust-lang/rust/issues/27709#issuecomment-231280999), ipv4Addr.is_unspecified shouldn't be unstable (though this needs a PR from me to mark it as stable).